### PR TITLE
10 create a method to set logging level

### PR DESCRIPTION
**Pull request by Adakar**

### CODE REVIEW

File: squape/report.py [hunk 1]
```diff
@@ -17,8 +17,44 @@ class LogLevel:
...
+
+
+def _translate_Level(level) -> int:
+    """Translates the given log level to valid LogLevel
+
+    Args:
+        level (int|str): log level to translate
+    """
+    if isinstance(level, int):
+        rv = level
```
Comment (pawtom) [inline squape/report.py:50]: Maybe its better to already retun here?
return level

File: squape/report.py [hunk 2]
```diff
@@ -17,8 +17,44 @@ class LogLevel:
...
+        "WARN": WARNING,
+        "WARNING": WARNING,
+        "LOG": LOG,
+        "DEBUG": DEBUG,
+    }
 
-LOGLEVEL = LogLevel.LOG
+
+def set_level(level) -> None:
+    """Sets the Squish logging level, Level must be an int or a str.
```
Comment (pawtom) [inline squape/report.py:31]: Please add examples for both int and str arguments.

File: squape/report.py [hunk 3]
```diff
@@ -17,8 +17,44 @@ class LogLevel:
...
+        level (int|str): log level to set
+    """
+    global LOGLEVEL
+    LOGLEVEL = _translate_Level(level)
+
+
+LOGLEVEL = set_level(LogLevel.LOG)
+
+
+def _translate_Level(level) -> int:
```
Comment (pawtom) [inline squape/report.py:43]: Why some interal module functions starts with _ and some with __?

**Review by pawtom — CHANGES_REQUESTED** [review]

File: squape/report.py (hunk 1 above)
Comment (Adakar) [reply squape/report.py:50]: In this case I just prefer having one return statement in the function

**Review by pawtom — APPROVED** [review]

**Pull request merged**